.github/workflows/lint-test-sdk.yml

```diff
@@ -21,6 +37,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
+          repository: Eppo-exp/sdk-common-jdk
```

Comment:
explicitly specify the repo and ref otherwise, when run as a reusable workflow in another repo, this would not checkout the java-common SDK as required